conf/ref_databases.config

```
@@ -70,12 +70,20 @@ params {
             dbversion = "GTDB R05-RS95 (https://data.ace.uq.edu.au/public/gtdb/data/releases/release95/95.0/)"
         }
         'pr2' {
-            title = "PR2 - Protist Reference Ribosomal Database - Version 4.14.0"
-            file = [ "https://github.com/pr2database/pr2database/releases/download/v4.14.0/pr2_version_4.14.0_SSU_dada2.fasta.gz", "https://github.com/pr2database/pr2database/releases/download/v4.14.0/pr2_version_4.14.0_SSU_UTAX.fasta.gz" ]
+            title = "PR2 - Protist Reference Ribosomal Database - Version 5.0.0"
+            file = [ "https://github.com/pr2database/pr2database/releases/download/v5.0.0/pr2_version_5.0.0_SSU_dada2.fasta.gz", "https://github.com/pr2database/pr2database/releases/download/v5.0.0/pr2_version_5.0.0_SSU_UTAX.fasta.gz" ]
             citation = "Guillou L, Bachar D, Audic S, Bass D, Berney C, Bittner L, Boutte C, Burgaud G, de Vargas C, Decelle J, Del Campo J, Dolan JR, Dunthorn M, Edvardsen B, Holzmann M, Kooistra WH, Lara E, Le Bescot N, Logares R, Mahé F, Massana R, Montresor M, Morard R, Not F, Pawlowski J, Probert I, Sauvadet AL, Siano R, Stoeck T, Vaulot D, Zimmermann P, Christen R. The Protist Ribosomal Reference database (PR2): a catalog of unicellular eukaryote small sub-unit rRNA sequences with curated taxonomy. Nucleic Acids Res. 2013 Jan;41(Database issue):D597-604. doi: 10.1093/nar/gks1160. Epub 2012 Nov 27. PMID: 23193267; PMCID: PMC3531120."
             fmtscript = "taxref_reformat_pr2.sh"
-            dbversion = "PR2 v4.14.0 (https://github.com/pr2database/pr2database/releases/tag/v4.14.0)"
-            taxlevels = "Domain,Kingdom,Phylum,Class,Order,Family,Genus,Species"
+            dbversion = "PR2 v5.0.0 (https://github.com/pr2database/pr2database/releases/tag/v5.0.0)"
+            taxlevels = "Domain,Clade,Kingdom,Phylum,Class,Order,Family,Genus,Species"
```

needs to be changed:
```suggestion
            taxlevels = "Domain,SuperClade,Clade,Phylum,Class,Order,Family,Genus,Species"
```

---

Hmm, looking closer at PR2, shouldn't it be "Domain (replacing Kingdom), Supergroup, Division, Subdivision (new taxonomic rank added), Class, Order, Family, Genus, and Species" as written in the PR2 release? @erikrikarddaniel? I feel a bit lost here...
In any case, it should of course be the same at both locations ("pr2" and "pr2=2.5.0")